cdc/owner/ddl_manager.go

```diff
@@ -130,6 +131,9 @@ type ddlManager struct {
 
 	BDRMode       bool
 	ddlResolvedTs model.Ts
+
+	sinkConfig  *config.SinkConfig
```

Comment:
We just use the m.sinkConfig.ShouldSendAllBootstrapAtStart() in ddlManager, could we just store this parameter, not all the sinkConfig.

It seems a little wired that ddlManager has a member call sinkConfig.